.github/workflows/test.yaml

```diff
@@ -99,6 +121,7 @@ jobs:
         run: |
           eval "$(conda shell.bash hook)"
           conda activate test-environment
+          python -m pip install --no-deps --no-build-isolation -e .
```

Comment:
Could you move this to its own step before the `doit env_capture` step? I like to have the `doit env_capture` step super simple, that's a least one step I know is not causing any issue when I debug the CI 🙃